lib/Dialect/SPIRV/Serialization/Serializer.cpp

```diff
@@ -411,6 +416,19 @@ LogicalResult Serializer::processTypeDecoration<spirv::ArrayType>(
   }
   return success();
 }
+
+LogicalResult
+Serializer::processMemberDecoration(uint32_t structID, uint32_t memberNum,
+                                    spirv::Decoration decorationType,
+                                    uint32_t value) {
+  SmallVector<uint32_t, 4> args;
```

Comment:
I think we can just do `... args({structID, ...})` to initialize directly?

Reply:
Thanks, I'll fix it!